Movie List/View Controllers/AddMovieViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AddMovieViewController.swift
+//  Movie List
+//
+//  Created by Peggy Wollenhaupt on 5/30/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+protocol AddMovieDelegate {
+    func movieWasAdded(_ movie: Movie)
+}
+
+class AddMovieViewController: UIViewController {
+
+    @IBOutlet weak var addMovieTextField: UITextField!
+    
+    var delegate: AddMovieDelegate?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func addMovieTapped(_ sender: Any) {
+        guard let title = addMovieTextField.text else { return }
+        
+        let movie = Movie(name: title)
+        delegate?.movieWasAdded(movie)
+        
+    }
```

Comment:
Absolute only thing I could offer as improvement is what we went over on the zoom 1 on 1... 
```suggestion
    @IBAction func addMovieTapped(_ sender: Any) {
        guard let title = addMovieTextField.text else { return }
        
        let movie = Movie(name: title)
        delegate?.movieWasAdded(movie)
        navigationController?.popViewController()
    }
```
This will pop the top view controller back to the previous view controller or the one right underneath the current one in the stack.